src/Components/WebAssembly/Server/src/WebAssemblyRazorComponentsBuilderExtensions.cs

```diff
@@ -48,4 +51,25 @@ public static IRazorComponentsBuilder AddAuthenticationStateSerialization(this I
 
         return builder;
     }
+
+    /// <summary>
+    /// Adds services to enforce Server culture on the Client side.
+    /// </summary>
+    /// <param name="builder">The <see cref="IRazorComponentsBuilder"/>.</param>
+    /// <returns>An <see cref="IRazorComponentsBuilder"/> that can be used to further customize the configuration.</returns>
+    public static IRazorComponentsBuilder EnforceServerCultureOnClient(this IRazorComponentsBuilder builder)
```

Comment:
We are going to need API review for this bit

Reply:
Make sure to file an issue for it. You can use the /ApiReview prompt to generate the contents. (Ask @oroztocil to show you how)